app/controllers/reviews_controller.rb

```diff
@@ -0,0 +1,23 @@
+class ReviewsController < ApplicationController
+  def new
+    @game = Game.find(params[:game_id])
+  end
+
+  def create
+    @game = Game.find(params[:game_id])
+    review = @game.reviews.create(review_params)
+    if review.save
+      flash[:success] = "Your Review Has Been Added"
+      redirect_to "/games/#{@game.id}"
+    else
+      flash[:error] = review.errors.full_messages.to_sentence
+      render :new
+    end
+  end
+
+  private
+
+  def review_params
+    params.permit(:title, :user_rating, :content)
```

Comment:
We could change it so that a user can add a review with just the rating but for now, this requirement works